modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/tracker/ObjectSizeCalculator.java

```diff
@@ -112,11 +98,10 @@ private static long calcAlign() {
 
         // Binary objects.
         // Assume objects is not deserialized.
-        addSysClsSize(BinaryObjectImpl.class, (c, bo) -> align(byteArrOffset + bo.array().length));
-        addSysClsSize(BinaryObjectOffheapImpl.class, null); // No extra heap memory.
```

Comment:
Can we just load class with ClassLoader. Pretty much code only for returning null.

Reply:
Yes, we can. But this will still keep link between `ObjectSizeCalculator` and binary in the form of class FQDN.
But, current version of PR reduce link to simple, understable methods.